### PR TITLE
Bonus: Make text post ornament transparent.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?

I was _just about_ to sneak this into #1302 but @mendelB beat me to it and reviewed the PR! Anyways, this pull request makes the "ornament" on the side of text posts transparent, as @lkpttn once only dreamt. Gaze upon it's technicolor beauty:

Screen Recording 2019-03-11 at 05 36 PM

One downside – now you can barely see it on normal campaign galleries:

Screen Shot 2019-03-11 at 5 40 45 PM

Well, you can't have it all. 🤷‍♂️ 

### Any background context you want to provide?

I guess we could apply a separate "static" color modifier on campaign pages? I dunno.

### What are the relevant tickets/cards?

N/A

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by DFurnes** [comment]
I added a rule to make these look a little less weird on "classic" campaign pages in a2c649c3:

Screen Shot 2019-03-12 at 10 55 01 AM

Since we're not using text galleries outside of the story page, this seems fine enough to ship.

**Comment by weerd** [comment]
This is rad 🎨 🎉

File: resources/assets/components/utilities/PostCard/post.scss [hunk 1]
```diff
@@ -27,8 +28,14 @@
   width: 100%;
 }
 
-.post-text {
-  border-left: $half-spacing solid #7069d8;
+.post-ornament-text {
+  border-left: $half-spacing solid rgba(#fff, 0.3);
```
Comment (weerd) [inline resources/assets/components/utilities/PostCard/post.scss]: woah, wait. You can write it as a hex value in the `rgba()` declaration?? This is sorcery!

File: resources/assets/components/utilities/PostCard/post.scss [hunk 2]
```diff
@@ -27,8 +28,14 @@
...
+  background-clip: padding-box; // Allow border to "see through" background.
```
Comment (weerd) [inline resources/assets/components/utilities/PostCard/post.scss]: Sneaky! 👌

File: resources/assets/components/utilities/PostCard/post.scss (hunk 1 above)
Comment (DFurnes) [reply resources/assets/components/utilities/PostCard/post.scss]: It's a Sass thing!! 👓 🙌

**Pull request merged**